main/provider/proxy.ts

```diff
@@ -9,10 +9,7 @@ class ProviderProxyConnection extends EventEmitter {
 
   async send(payload: JSONRPCRequestPayload) {
     if (payload.method === 'eth_subscribe') {
-      const { id, jsonrpc, params } = payload
-      const subType = params[0] as string
-
-      this.emit('provider:subscribe', { id, jsonrpc, params: [subType] })
+      this.emit('provider:subscribe', payload)
```

Comment:
Can we guarantee that params here only has one element?  Just thinking this code could have been a weird way to strip out params after the first one

Reply:
we want to pass the entire payload on regardless of how many params there are, we just want to route it to a different place so that the provider creates the internal subscription correctly